packages/flutter/lib/src/material/tabs.dart

```diff
@@ -303,7 +303,7 @@ class _IndicatorPainter extends CustomPainter {
 
   static bool _tabOffsetsNotEqual(List<double> a, List<double> b) {
     assert(a != null && b != null && a.length == b.length);
-    for(int i = 0; i < a.length; i++) {
+    for (int i = 0; i < a.length; i += 1) {
```

Comment:
Is `++` considered bad and should be avoided?

Reply:
No I just have a pet peeve against it. :-)